e2e/davinci-app/components/flow-link.ts

```diff
@@ -15,7 +15,7 @@ export default function flowLinkComponent(
   formEl?.appendChild(button);
 
   button.addEventListener('click', async () => {
-    const node = await flow(collector.output.key);
-    renderForm(node);
+    await flow(collector.output.key);
```

Comment:
Just something that crossed my mind, the word `flow` may be too vague of a term in the context of a larger codebase. Maybe we should rename it to `davinciFlow` or `authFlow` or something more descriptive that won't get lost inside a larger system.

Reply:
Maybe not, maybe its just in this app how I read the param/variable here

Reply:
Oh, yeah. This is just a parameter that holds the returned function from calling `davinciClient.flow()`. It could be named anything. Its value comes from here: https://github.com/ForgeRock/forgerock-javascript-sdk/blob/davinci-client/e2e/davinci-app/main.ts#L82.